flink-table/flink-table-planner/src/test/resources/jsonplan/testGetJsonPlan.out

```diff
@@ -1,76 +1,110 @@
 {
-   "flinkVersion":"",
-   "nodes":[
-      {
-         "class":"org.apache.flink.table.planner.plan.nodes.exec.stream.StreamExecTableSourceScan",
-         "scanTableSource":{
-            "identifier":{
-               "catalogName":"default_catalog",
-               "databaseName":"default_database",
-               "tableName":"MyTable"
+  "flinkVersion": "",
+  "nodes": [
+    {
+      "class": "org.apache.flink.table.planner.plan.nodes.exec.stream.StreamExecTableSourceScan",
+      "scanTableSource": {
+        "catalogTable": {
+          "identifier": "`default_catalog`.`default_database`.`MyTable`",
+          "catalogTable": {
+            "resolvedSchema": {
+              "columns": [
+                {
+                  "name": "a",
+                  "type": "physical",
+                  "dataType": "BIGINT"
+                },
+                {
+                  "name": "b",
+                  "type": "physical",
+                  "dataType": "INT"
+                },
+                {
+                  "name": "c",
+                  "type": "physical",
+                  "dataType": {
+                    "logicalType": "VARCHAR(2147483647)",
```

Comment:
let's only serialize the logical type

Reply:
This is done here https://github.com/apache/flink/pull/18506